src/components/security/AuthTokenList.vue

```diff
@@ -74,21 +74,39 @@ export default defineComponent({
 
 <style lang="scss" scoped>
 .token-list {
-	width: 100%;
+	width: 66.6%;
```

Comment:
```suggestion
```
* let's remove with in oder not to stretch table in big screens and keep table small